.eslintrc.json

```diff
@@ -0,0 +1,214 @@
+{   "root": true,
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "globals" : {
+        "FontFaceObserver": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true,
+            "jsx": false
+        },
+        "sourceType": "module"
+    },
+    "plugins": [
+
+    ],
+    "rules": {
+        // Possible Errors (overrides from recommended set)
```

Comment:
Wait. Is this a comment in JSON!? pretty sure that's invalid.

Reply:
Yep, so found two ways to write comments in json, but without the syntax color change for comments it might not be as helpful. The comments does help to break up each chunk of the eslint rules..thoughts?

https://stackoverflow.com/questions/244777/can-comments-be-used-in-json
`"___comment":  "Comments with // or /**/ are NEVER used in a JSON file!!!",`

https://www.quora.com/How-do-I-write-comments-inside-a-JSON-document
`  “///////”: “This is a comment for actualKey”,`

Reply:
See new .eslintrc.json file, regular comments like so:
`"////": "_____//Enforces the style of wrapped functions",`

section comments like so preceded by two line returns
`"////": "_____VARIABLES_____",`

Hopefully this adds value.